.github/workflows/types.yml

```diff
@@ -1,12 +1,15 @@
 name: Types
 
 on:
-  pull_request:
   push:
-    # filtering branches here prevents duplicate builds from pull_request and push
     branches:
       - main
       - master
+  pull_request: {}
```

Comment:
Hmm, what's the status on this particular file, @SergeAstapov? Just stumped b/c of what it was doing before?

Reply:
@chriskrycho TBH not sure we need this file as workflow is commented out and `tsc` is part of `ci` workflow.

I think it worth deleting it in separate PR.

Here, I tried to keep it in sync with tweaks to ci.yml

Reply:
Seems good. Let's land it as is and delete in a separate PR. :)